go/fn/testhelpers/golden.go

```diff
@@ -251,7 +262,11 @@ func CompareFile(t *testing.T, expectDir, actualDir string, relPath string) {
 }
 
 func mustParseFile(t *testing.T, path string) fn.KubeObjects {
+	if _, err := os.Stat(path); errors.Is(err, os.ErrNotExist) {
```

Comment:
This handles the case that "_fnconfig.yaml" is not provided (when the function does not need a fnConfig like `kubeval` `set-standard-labels`